contracts/staking/StakeManager.sol

```diff
@@ -144,7 +185,16 @@ contract StakeManager is Validator, IStakeManager, RootChainable, Lockable {
       auction.amount = amount;
       auction.user = msg.sender;
     }
-    emit StartAuction(validatorId, validators[validatorId].amount, validatorAuction[validatorId].amount);
+    logger.logStartAuction(validatorId, validators[validatorId].amount, validatorAuction[validatorId].amount);
+  }
+
+  function cancelAuctionBid(uint256 validatorId) public {
```

Comment:
yeah that's why `require(auction.user != ownerOf(validatorId));` but i still need to think upon how to prevent this mech. In any case, people need to invest capital for 1-2 week which is a loss for them. Only person who is investing less is the Owner of the current slot so in that case, I am thinking of forcefully confirming that.